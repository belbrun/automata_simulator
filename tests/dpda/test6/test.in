0,1,1,2
qp1,q0,qpr
0,1,2
K,N,J
qpr
qp1
K
qp1,0,K->q0,NK
q0,0,N->q0,NN
q0,0,J->q0,$
qp1,1,K->q0,JK
q0,1,N->q0,$
q0,1,J->q0,JJ
qp1,2,K->q0,K
q0,2,N->q0,N
q0,2,J->q0,J
q0,$,K->qp1,K
