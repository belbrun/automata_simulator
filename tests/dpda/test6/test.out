qp1#K|q0#NK|q0#K|qp1#K|q0#JK|q0#JK|0
